src/views/app/detail-panel/preview/mail-preview.jsx

```diff
@@ -121,6 +117,8 @@ const MailPreviewBlock = ({ message, open, onClick }) => {
 			? { color: 'text', weight: 'regular', badge: 'read', size: 'small' }
 			: { color: 'primary', weight: 'bold', badge: 'unread', size: 'medium' };
 	}, [message.read]);
+	const actions = useMessageActions(message);
+
 	return (
 		<>
 			{folderId === '4' && (
```

Comment:
Can we replace this with FOLDERS.DRAFTS ?

Reply:
Everything related to code looks correct to me the only thing that doesn't look good is the UI when we have long string of 'on behalf of'
![behalf](https://user-images.githubusercontent.com/74703491/153153689-954cd4b3-bc9a-4fef-af18-f83ab23f264e.png)

is it planned for another ticket ?

@zovomat @giuliano176   can we replace 
`								<Text overflow="break-word">
									<Text
										data-testid="SenderText"
										size={message.read ? 'small' : 'medium'}
										color={message.read ? 'text' : 'primary'}
										weight={message.read ? 'normal' : 'bold'}
									>
										{senderContact.fullName && `"${senderContact.fullName}"`}
									</Text>
									<Padding left="small">
										<Text color="gray1" size={message.read ? 'small' : 'medium'}>
											{senderContact.address && `<${senderContact.address}>`}
										</Text>
									</Padding>
									<Text size="small">{` ${t('label.on_behalf_of', 'on behalf of')} `}</Text>
									<Text
										data-testid="SenderText"
										size={message.read ? 'small' : 'medium'}
										color={message.read ? 'text' : 'primary'}
										weight={message.read ? 'normal' : 'bold'}
									>
										{mainContact.fullName && `"${mainContact.fullName}"`}
									</Text>

									<Padding left="small">
										<Text color="gray1" size={message.read ? 'small' : 'medium'}>
											{mainContact.address && `<${mainContact.address}>`}
										</Text>
									</Padding>
								</Text>`  with something like this

`<Trans
									i18nKey="label.behalf_of"
									values={{
										fullName: senderContact.fullName,
										address: senderContact.address,
										mainContactFullName: mainContact.fullName || mainContact.name,
										mainContactAddress: mainContact.address
									}}
									defaults={
										'<Row> <PrimaryText> "{{fullName}}" </PrimaryText>  <GrayText> {{address}} </GrayText> <NormalText> on behalf of </NormalText> <PrimaryText> "{{mainContactFullName}}" </PrimaryText> <GrayText> <{{mainContactAddress}}> </GrayText> </Row>'
									}
									components={{
										Row: <TextWithTooltip />,
										PrimaryText: (
											<span
												style={{
													color: message.read ? '#333333' : '#2b73d2',
													fontWeight: message.read ? 'normal' : 'bold'
												}}
											/>
										),
										GrayText: (
											<span
												style={{ color: '#828282', fontWeight: message.read ? 'normal' : 'bold' }}
											/>
										),
										NormalText: <span />
									}}
								/>`

Reply:
![Screenshot 2022-02-09 at 1 33 31 PM](https://user-images.githubusercontent.com/74703491/153155112-9a2b1703-bdf0-4757-b0ac-2530b5f117c2.png)
the above code will give this result

Reply:
I would say to open a different task for it as I am already overtime by more than 30% of the estimated time